### ISSUE TITLE
Oxfordshire localisation – Prove vehicle ownership

### ISSUE THREAD

**Issue by suryanaga** [alphagov/verify-local-patterns#659]
1. Add an extra page for company cars

 Page asks - "Do you own the vehicle or is it a company car?"

Page URL - /vehicle-ownership
Page before - /confirm-address
Page after if yes - /v5c-upload (existing page)
Page after if no - /company-car-upload (new page)

2. Make more clear on start page you will need this document

**Comment by suryanaga** [comment]
@abibarkeroxfordshire Could you add your policy regarding vehicle ownership to this issue?

**Comment by AbiBarkerOxfordshire** [comment]
Here you go @sanjaypoyzer : (it's not exactly snappy....)

Full time employees may obtain a permit for a vehicle provided by their employers. To obtain a
permit a letter must be produced on company headed paper in the following format. This to
confirm that “applicant’s name” is a full time employee of “company name” and is supplied with
“vehicle make/model/registration number” for his/her sole business and social use. The vehicle
is used essentially for the operational purpose of conducting official company business from the
Oxford area.
This letter MUST be produced with EITHER a copy of the registration document (if the vehicle is
registered in the name of the Company) or a copy of the vehicle lease agreement between the
company and the vehicle provider, TOGETHER with recent official proof of residency.

**Comment by suryanaga** [comment]
Thanks!

**Comment by suryanaga** [comment]
@abibarkeroxfordshire Hi, can you clarify, it seems to me from that that people using company cars have to provide a V5C ("registration document") as well as a letter from the company authorising usage?

**Comment by AbiBarkerOxfordshire** [comment]
@sanjaypoyzer Yep - they need to provide the company letter plus EITHER the V5C or lease agreement to confirm the company owns or leases the vehicle.

**Comment by suryanaga** [comment]
Ok cool. 

@abibarkeroxfordshire - One more question, can users who aren't using a company car also provide a lease agreement if they are renting a car?

@lizziebruce - We're adding new pages for vehicle ownership, can you suggest some content?

So after the user is verified, they are asked something like: 

> ## Is the vehicle registered in your name or registered to your employer?
> - It's my car
> - It's my employer's car

If it's their car, they are routed to upload V5C. We already have a page for this.

If it's not, they have to upload a V5C _or_ car lease agreement registered to their employer _and_ upload a letter from their employer authorising usage. (See policy above for the exact rules on this letter.)

We'll also need content explaining all this for the start page. Thanks!

**Comment by AbiBarkerOxfordshire** [comment]
@sanjaypoyzer Yes they can - the other rules for leases and new cars are:

Vehicles rented through a reputable firm for a minimum of one year will qualify for a parking
permit however borrowed vehicles will not.

For residents who have purchased a vehicle from a dealer within the last month can produce a
sales invoice showing their name to qualify for a permit.

**Comment by lizziebruce** [comment]
How about:

> **Is the vehicle registered in your name?**
> 
> - Yes 
> - No

Yes - continue as usual to VC5 page
No goes to

**Vehicle ownership**

> The vehicle is provided by:
> 
> - my employer
> - a rental firm that I've hired it from
> - myself but it's not yet registered
> 
If they choose employer, next page is a) below.

If they choose a rental firm that I've hired it from, next page is b) below.

If they choose myself but it's not yet registered, next page is c) below.

a)
> **Vehicle ownership document**
> 
> First upload 1 of the following:
> 
> - V5C document for the car
> - lease agreement registered to your employer
>
> [UPLOAD] 

>**Letter from your employer**
>
> Now upload a letter from your employer on company stationary allowing you to use the car:
>
> [UPLOAD]

b)
> **Rented vehicle**
> 
> Upload the lease agreement for the vehicle registered to yourself:
> 
> [UPLOAD]

c) 
> **New vehicle**
> 
> Upload the sales invoice for your new vehicle, showing your name:
>
> [UPLOAD]

**Comment by suryanaga** [comment]
@lizziebruce Sounds good to me! 

@AbiBarkerOxfordshire What do you think?
